src/components/activity-page/managed-interactive/iframe-runtime.tsx

```diff
@@ -139,6 +148,20 @@ export const IframeRuntime: React.ForwardRefExoticComponent<IProps> = forwardRef
       addListener("hint", (newHint: any) => {
         setNewHint(newHint.text || "");
       });
+      addListener("decoratedContentEvent", (msg: IDecoratedContentEventMessage) => {
+        if (textDecorationInfo && msg.type === "click") {
+          if ("type" in textDecorationInfo.eventListeners && textDecorationInfo.eventListeners.type === "click") {
```

Comment:
Any reason not to just pass the `type` through here? So if new types are added this code doesn't need to be updated.

Reply:
Yes, good point.  As long as the `type` in the message matches a `type` in our stored listener, then we can pass it on.  Fixed.